servers/mu/src/domain/index.js

```diff
@@ -27,7 +27,7 @@ import { createLogger } from './logger.js'
 import { cuFetchWithCache } from './lib/cu-fetch-with-cache.js'
 import { handleWorkerQueueMessage } from './lib/handle-worker-queue-message.js'
 import { BroadcastChannel } from 'node:worker_threads'
-
+import { apply } from 'ramda'
```

Comment:
Could we fix the order of these import statements? `node` imports at the top, then imports from `node_modules`, then locals